zeroconf/__init__.py

```diff
@@ -229,7 +236,7 @@ def _encode_address(address: str) -> bytes:
     return socket.inet_pton(address_family, address)
 
 
-def service_type_name(type_: str, *, allow_underscores: bool = False) -> str:
+def service_type_name(type_: str, *, allow_underscores: bool = False, strict: bool = True) -> str:
```

Comment:
`ServiceBrowser` wants `strict` and `allow_underscores` currently.  I'll check to see if it really need `strict` as it may already be checked before that anyways

Reply:
It looks like I can't combine them without a breaking change in `ServiceBrowser`